app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -102,3 +102,48 @@ def test_check_with_rapid_review_and_no_kinto_pending_pushes_experiment(self):
         self.assertEqual(experiment.firefox_min_version, Experiment.VERSION_CHOICES[0][0])
         self.assertEqual(experiment.firefox_channel, Experiment.CHANNEL_RELEASE)
         self.assertEqual(experiment.normandy_slug, "bug-12345-rapid-test-release-55")
+
+
+class TestCheckExperimentIsLive(MockKintoClientMixin, TestCase):
+    def test_experiment_updates_when_recipe_is_in_main(self):
```

Comment:
We coulda stuck a draft in here just to make sure it DOESN'T get included in the check, but that's just a nice to have, not a blocker

Reply:
Alright let's just add it